frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions/ExpressionEditorSuggestions.tsx

```diff
@@ -87,7 +95,7 @@ export function ExpressionEditorSuggestions({
   const groups = group(items);
 
   function handleMouseDown(evt: MouseEvent) {
-    if (evt.target === ref.current) {
+    if (evt.target === mergedRef.current) {
```

Comment:
`mergedRef` returns a function so we cannot read the `.current` property, you will need to read from the `listRef` instead.

```suggestion
    if (evt.target === listRef.current) {
```